package.json

```diff
@@ -5,15 +5,18 @@
   "main": ".eslintrc.js",
   "scripts": {
     "prebuild": "rimraf _site",
-    "build": "cross-env ELEVENTY_ENV=production run-s build:*",
+    "build": "cross-env ELEVENTY_ENV=production run-s build:* && move:*",
     "build:eleventy": "eleventy",
     "build:js": "rollup --config",
     "build:css": "node-sass --importer node_modules/node-sass-glob-importer/dist/cli.js src/assets/scss/app.scss _site/styles/index.css",
+    "premove:fonts": "cross-env ELEVENTY_ENV=development rimraf _site/fonts",
```

Comment:
You don't need any of these scripts. What you instead can do is leverage the eleventyConfig file and use [passthroughCopy](https://www.11ty.dev/docs/copy/). This way fonts get copied at build time and watched over at dev.